src/Wizard/Wizard.jsx

```diff
@@ -42,11 +42,19 @@ export class Wizard extends React.Component {
   }
 
   nextStepHandler() {
-    if (this.state.currentStep === this.props.steps.length - 1) {
-      this.props.onComplete(this.state.data);
+    const {steps, onComplete} = this.props;
+    const {currentStep, data} = this.state;
+
+    // If an onComplete handler is provided for the current step, and it returns false, we won't
+    // proceed to the next step.
+    if (steps[currentStep].onComplete && !steps[currentStep].onComplete(data)) {
```

Comment:
From a usage perspective, it seems like it might be good to take an error message string as the response of onStepComplete and then above the buttons populate an a alert (which is not part of Clever components?)

`errorMsgStr = onStepComplete(data)`

What are your thoughts on this?